### PR TITLE
fix: remove unused delegation field

### PR DESCRIPTION
**Pull request by wa0x6e**
The `delegation` column in the proposals table is unused, and should be removed.

Should run this query to drop the column 

```sql
ALTER TABLE proposals DROP COLUMN delegation;
```

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]
utAck

File: test/schema.sql [hunk 1]
```diff
@@ -32,7 +32,6 @@ CREATE TABLE proposals (
   choices JSON NOT NULL,
   start INT(11) NOT NULL,
   end INT(11) NOT NULL,
-  delegation INT(1) NOT NULL,
```
Comment (ChaituVR) [inline test/schema.sql:35]: Is there some other way to use schema from hub 🙈  we didn't add many fields here from there recently 🙈

Comment (wa0x6e) [reply test/schema.sql:35]: What you have in mind ? github actions to sync the file with hub ?

Comment (ChaituVR) [reply test/schema.sql:35]: Excellent Idea! I think we can check for changes on each pull request sequencer and sync with the hub

**Pull request merged**